eva_submission/nextflow/accession_and_load.nf

```diff
@@ -323,37 +324,103 @@ process load_vcf {
     script:
     def pipeline_parameters = ""
 
-    if (params.annotation_only) {
-        pipeline_parameters += " --spring.batch.job.names=annotate-variants-job"
-    } else if(aggregation.toString() == "none"){
-        pipeline_parameters += " --spring.batch.job.names=genotyped-vcf-job"
+    if(aggregation.toString() == "none"){
+        pipeline_parameters += " --spring.batch.job.names=genotyped-variant-load-vcf-job"
     } else{
-        pipeline_parameters += " --spring.batch.job.names=aggregated-vcf-job"
+        pipeline_parameters += " --spring.batch.job.names=aggregated-variant-load-vcf-job"
     }
 
     pipeline_parameters += " --input.vcf.aggregation=" + aggregation.toString().toUpperCase()
     pipeline_parameters += " --input.vcf=" + vcf_file.toRealPath().toString()
     pipeline_parameters += " --input.vcf.id=" + analysis_accession.toString()
     pipeline_parameters += " --input.fasta=" + fasta.toString()
-
     pipeline_parameters += " --spring.data.mongodb.database=" + db_name.toString()
 
-    if (vep_version.trim() == "" || vep_cache_version.trim() == "") {
-        pipeline_parameters += " --annotation.skip=true"
-    } else {
-        pipeline_parameters += " --annotation.skip=false"
-        pipeline_parameters += " --app.vep.version=" + vep_version.toString()
-        pipeline_parameters += " --app.vep.path=" + "${params.vep_path}/ensembl-vep-release-${vep_version}/vep"
-        pipeline_parameters += " --app.vep.cache.version=" + vep_cache_version.toString()
-        pipeline_parameters += " --app.vep.cache.species=" + vep_species.toString()
+    """
+    java -Xmx4G -jar $params.jar.eva_pipeline --spring.config.location=file:$params.load_job_props --parameters.path=$params.load_job_props $pipeline_parameters
+    """
+}
+
+
+/*
+ * Load into variant db.
+ */
+process run_vep_on_variants {
+    clusterOptions {
+        return "-o $params.logs_dir/annotation.${vcf_filename}.log \
+                -e $params.logs_dir/annotation.${vcf_filename}.err"
     }
 
+    input:
+    tuple val(vcf_filename), val(vcf_file), val(fasta), val(analysis_accession), val(db_name), val(vep_version), val(vep_cache_version), val(vep_species), val(aggregation)
+    val variant_load_complete
+
+    output:
+    val true, emit: vep_run_complete
```

Comment:
I don't think the outputs for vep and stats are needed right now, though I guess they don't hurt...